### PR TITLE
Editorial changes

### PR DESCRIPTION
**Pull request by cyrichardson**
@catlook Please review these changes and see if you agree with them. Thanks!

### CODE REVIEW

File: _posts/2020-02-14-how-salesforces-2018-acquisitions-and-partnerships-change-how-you-work.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ which Salesforce customers can leverage alongside
 #### The benefits
 
 The partnership between these tech giants brings benefits for loyal fans of both
-brands, which audiences saw firsthand this year at Google's annual developer
+brands. Audiences saw it firsthand this year at Google's annual developer
```
Comment (catlook) [inline _posts/2020-02-14-how-salesforces-2018-acquisitions-and-partnerships-change-how-you-work.md]: ... brands. Audiences saw it firsthand at Google's annual developer ...

Comment (cyrichardson) [reply _posts/2020-02-14-how-salesforces-2018-acquisitions-and-partnerships-change-how-you-work.md]: Updated!

**Comment by rackernexus** [comment]
Build 750 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-1557c6fb8/). valid for 14 days

**Comment by rackernexus** [comment]
Build 752 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-499118b82/). valid for 14 days

**Pull request merged**